pkg/provider/gitlab/gitlab.go

```diff
@@ -174,7 +174,12 @@ func (g *Gitlab) GetSecret(ctx context.Context, ref esv1beta1.ExternalSecretData
 	// 	"value": "TEST_1",
 	// 	"protected": false,
 	// 	"masked": true
```

Comment:
Maybe you could extend the sample payload so that it shows the actual "environment_scope" that is being returned by gitlab.